src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1728,15 +1728,14 @@ __meta__ = {
 [node name="Value" type="Label" parent="MicrobeHUD/BottomRight/PopulationData"]
 margin_left = 86.0
 margin_top = 8.0
-margin_right = 121.0
+margin_right = 125.0
 margin_bottom = 25.0
 rect_min_size = Vector2( 35, 0 )
 size_flags_horizontal = 3
 custom_styles/normal = SubResource( 28 )
 custom_fonts/font = ExtResource( 12 )
 text = "100"
 align = 1
-autowrap = true
```

Comment:
Unintended change? Doesn't seem to fit this PR.

Reply:
It's an extra fix for the species pop number in the microbe HUD to not autowrap (which is not needed anymore now the numbers are formatted with suffixes)